src/OpenLoco/src/Windows/BuildVehicle.cpp

```diff
@@ -226,6 +248,29 @@ namespace OpenLoco::Ui::Windows::BuildVehicle
         return widgetIndex - widx::tab_track_type_0;
     }
 
+    enum class VehicleFilterFlags : uint8_t
+    {
+        none = 0,
+        powered = 1 << 0,
+        unpowered = 1 << 1,
+        locked = 1 << 2,
```

Comment:
how does this play with the locked vehicles cheats? if we are now allowing display of locked vehicles without the cheat, we should just remove the cheat for display, and leave the one for building locked vehicles?

Reply:
These options should only show up when the cheat is active :)